clients/cli/src/orchestrator_client.rs

```diff
@@ -37,7 +37,8 @@ impl OrchestratorClient {
         U: Message + Default,
     {
         let request_bytes = request_data.encode_to_vec();
-        let url = format!("{}/v2{}", self.base_url, url);
+        let api_version = "v3";
```

Comment:
why bother extracting this? I don't see it changed anywhere after it's first use.

Reply:
It's sort of a magic constant and it usually improves readability to name magic constants. I'm happy to change it back though.